src/app/auth/auth.routing.ts

```diff
@@ -0,0 +1,18 @@
+import { Routes, RouterModule } from '@angular/router';
+import { NgModule } from '@angular/core';
+
+
+import { LoginComponent } from './login/login.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7127387)
```suggestion
import { LoginComponent } from "./login/login.component";
```